### ISSUE TITLE
Docker is not configured correctly

### ISSUE THREAD

**Issue by schultyy** [travis-ci/travis-enterprise-worker-installers#13]
`export TRAVIS_WORKER_DOCKER_ENDPOINT='tcp://localhost:4243'` is not working right now, since Docker is configured to launch with `fd://`.

**Comment by schultyy** [comment]
The Docker endpoint has to be configured like this:

```
export TRAVIS_WORKER_DOCKER_ENDPOINT='unix:///var/run/docker.sock'
```

**Comment by schultyy** [comment]
Resolved in #16
